package.json

```diff
@@ -18,14 +19,26 @@
     "plugin"
   ],
   "author": "Nicolas Herment <nherment@gmail.com>",
+  "contributors": [
+    {
+      "name": "Matthew O'Connor",
+      "url": "https://github.com/matthewoconnor-development"
+    }
+  ],
   "license": "MIT",
   "bugs": {
     "url": "https://linkedin.com/nherment/seneca-linkedin-auth/issues"
   },
   "homepage": "https://linkedin.com/nherment/seneca-linkedin-auth",
   "dependencies": {
-    "passport-linkedin": "0.1.3"
+    "passport-linkedin": "1.0.0"
   },
+  "devDependencies": {
+     "eslint-config-seneca": "1.1.x",
```

Comment:
also you need all these three plugins installed: 
eslint-config-seneca
eslint-plugin-standard
eslint-plugin-hapi

after you install them, please run the linter again, you might find more linting issues